src/Integration/LocalServices/ProjectSystemFilter.cs

```diff
@@ -62,10 +66,11 @@ public bool IsAccepted(DteProject project)
 
         #region Helpers
 
-        private static bool IsNotSupportedProject(DteProject project)
+        private bool IsNotSupportedProject(DteProject project)
         {
-            var language = ProjectToLanguageMapper.GetLanguageForProject(project);
-            return (language == null || !language.IsSupported);
+            var languages = projectToLanguageMapper.GetAllBindingLanguagesForProject(project);
+
+            return languages.Any(x => !x.IsSupported);
```

Comment:
Is this logic correct? It's saying _"the project is unsupported if any of the languages are not supported"._

Shouldn't it be _"the project is supported if any of the languages is supported"_ (i.e. it's only unsupported if all languages are unsupported)?

If the logic is wrong, then it suggests there's a missing test case.